### PR TITLE
Django3 and Django4 upgrade

### PR DESCRIPTION
**Pull request by CSprat451**
Based on Pull Request #59, upgraded to Django 3.0 and Django 4.0 support. Removed support for Django version 1.8, 1.9, and 1.10 and removed support for Python 2.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,23 +1,24 @@
 [tox]
 envlist =
-  {py27,py34}-dj{18,19,110}
-  {py27,py34,py35,py36}-dj111
-  {py34,py35,py36}-dj20
-  {py35,py36}-djdev
+  {py35,py36}-dj111
+  {py35,py36,py37,py38}-dj20
+  py39-dj{22,30,31,32,40}
```
Comment (bgaudino) [inline tox.ini:5]: Is there a reason that you test Django 2.0, but not 2.1?

Comment (CSprat451) [reply tox.ini:5]: Nope, added it in as an additional test.

File: betterforms/changelist.py [hunk 2]
```diff
@@ -101,7 +101,7 @@ def get_queryset(self):
         return qs
 
 
-class BoundHeader(object):
+class BoundHeader():
```
Comment (nolsto) [inline betterforms/changelist.py:104]: This and all other base classes that implicitly subclass `object` should not have parens. (`class BoundHeader:`).

File: tests/forms.py [hunk 3]
```diff
@@ -179,13 +179,13 @@ def clean(self):
 
 class RaisesErrorCustomCleanMultiform(UserProfileMultiForm):
     def clean(self):
-        cleaned_data = super(UserProfileMultiForm, self).clean()
+        cleaned_data = super().clean()
```
Comment (nolsto) [inline tests/forms.py:182]: This super call should be left  as was: `super(UserProfileMultiForm, self).clean()`.
Notice that the original code uses `UserProfileMultiForm` as the first argument to `super` even though the class in which the method is defined is `RaisesErrorCustomCleanMultiform`. This is not the same as the behavior of `super()` when the arguments are omitted.

File: tests/forms.py [hunk 4]
```diff
@@ -179,13 +179,13 @@ def clean(self):
...
         raise ValidationError('It broke')
         return cleaned_data
 
 
 class ModifiesDataCustomCleanMultiform(UserProfileMultiForm):
     def clean(self):
-        cleaned_data = super(UserProfileMultiForm, self).clean()
+        cleaned_data = super().clean()
```
Comment (nolsto) [inline tests/forms.py:189]: See comment on L182 above

File: CHANGES [hunk 5]
```diff
@@ -1,12 +1,17 @@
...
+Backwards-incompatible changes:
+
+- Removed support for python 2.7
+- Removed support for Django versions 1.8, 1.9, and 1.10.
 
-- Add support for Django 2.2, 3.0, 3.1, 3.2, and 4.0.
+New Features and Bugfixes:
+
+- Add support for Django 2.1, 2.2, 3.0, 3.1, 3.2, and 4.0.
 - Add support for Python 3.7, 3.8, and 3.9.
```
Comment (nolsto) [inline CHANGES:11]: python 3.10 is also in the tox.ini and setup.py classifiers.

File: betterforms/forms.py [hunk 6]
```diff
@@ -1,22 +1,25 @@
-import collections
+try:
```
Comment (gavinwahl) [inline betterforms/forms.py:1]: collections and collections.abc are not the same thing.

```
try:
    from collections.abc import Iterable
except ImportError:
    # BBB: Python < whatever
    from collections import iterable

**Pull request merged**